lib/msf/core/exceptions.rb

```diff
@@ -48,10 +48,22 @@ def to_s
   end
 end
 
+
+###
+#
+# This exception is raised when a module fails to load.
+#
+# It is used by Msf::Modules::Loader::Base.
+#
 ###
```

Comment:
cool, I think this could be useful in the golang support as well to bubble up specific module load issues

Reply:
We can flesh it out with additional methods later, but for now the message is the developer's concern.